### PR TITLE
Fix auth callback url on README

### PR DESCRIPTION
**Pull request by tsleal29**
- Fix authorization callback url
- Update to-do list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,11 +53,8 @@ Looking for inspiration on how to contribute?
 
 - use pundit to manage access rights
 - setup a CI (maybe using GitHub actions)
-- rank mentors by number of mentees
-- add location and language to users and show it on the mentors tab
```
Comment (fwuensche) [inline README.md:57]: This should only be removed in the PR where you implement the feature, no? But no worries, we'll wait to merge this PR last then 👌

Comment (tsleal29) [reply README.md:57]: Makes sense. My bad

Comment (fwuensche) [reply README.md:57]: So are you removing it?

Comment (tsleal29) [reply README.md]: I understood we were going to leave it and approve this PR after the others. But it's done. Moved the updates in the todo lists to the PR where they're implemented 👍

**Pull request merged**